### PR TITLE
CB-134 More information about Spotify albums on mapping page is shown now.

### PR DESCRIPTION
**Pull request by stasszczesniak**
CritiqueBrainz provides an embedded Spotify player on pages that are mapped to it in mbspotify. Mappings are created by users on CritiqueBrainz using a search interface that we have. Search gets data from Spotify using their Web API, but the only parts we use are cover art and name of the album. Now users can also see the artists that made some particular album and the release year.

[Related ticket] (http://tickets.musicbrainz.org/browse/CB-134)

Prototypes: 

1. **Artists from tracks if unavailable, date in the same line as artists and in brackets.**
data_w_nawiasie_przy_artystach_artysci_ze_sciezek_male

2. **Artists from default artists, date in the same line as artists and in brackets.**
data_w_nawiasie_przy_artystach_male

3. **Artists from tracks if unavailable, date in the same line as type and in brackets.**
data_w_nawiasie_przy_typie_artysci_ze_sciezek

4. **Artists from default artists, date in the same line as type and in brackets.**
data_w_nawiasie_przy_typie

5. **Artists from default artists, date in the next line.**
data_na_dole_various

6. **Artists from tracks if unavailable, date in the next line.**
data_na_dole_sciezki

7. **All artists from tracks if unavailable, date in the next line**
all_artists_from_tracks

8. **Freso's idea**
freso

### CODE REVIEW

**Comment by Freso** [comment]
Okay, this is crazy, but how about something like
"**Album Name**
by **Artist Name**
**2009** single
*Listen on Spotify*"
?

**Comment by stasszczesniak** [comment]
Added the #8 option regarding Freso's proposition.

**Comment by Freso** [comment]
I actually like that. It both visually expresses the difference, but at the same time also makes sense (e.g., »that's Iggy Pop's 2006 album«).

**Comment by gentlecat** [comment]
Yeah, I agree, that looks good. Is it implemented here or do you need to make changes?

File: critiquebrainz/frontend/templates/mapping/spotify.html [hunk 1]
```diff
@@ -29,9 +29,16 @@
           </a>
         </div>
         <div class="caption">
-          <div class="title">{{ item.name }}</div>
-          <span class="text-muted">{{ item.album_type }}</span><br />
-          <em><a href="{{ item.external_urls.spotify }}">{{ _('Listen on Spotify') }}</a></em><br />
+            <div class="title">
+                {{ item.name }}
+                <br>
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/mapping/spotify.html]: <br />

File: critiquebrainz/frontend/templates/mapping/spotify.html [hunk 2]
```diff
@@ -29,9 +29,16 @@
...
+                <span style="font-weight:normal">by </span>
+                {{ artists(item) }}
+                <br>
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/mapping/spotify.html]: Same here.

File: critiquebrainz/frontend/templates/mapping/spotify.html [hunk 3]
```diff
@@ -29,9 +29,16 @@
...
+            </div>
+            <b> {{ release_date_string(item) }} </b>
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/mapping/spotify.html]: There should be no spaces after opening and before closing tags.

**Comment by gentlecat** [comment]
You need to fix indentation in the template. It needs to be 2 spaces.

**Comment by stasszczesniak** [comment]
Should be fine now.

**Pull request merged**